getting-started/deployment_guide.md

```diff
@@ -36,42 +38,618 @@ KubeArmor Policy Manager Deployment ...
 KubeArmor Host Policy Manager Service ...
 KubeArmor Host Policy Manager Deployment ...
 ```
+
+</details>
+
+### 3. Deploying sample app
+
+#### a. Deploy sample [wordpress-mysql app](../examples/wordpress-mysql.md)
+
+```
+$ kubectl apply -f https://raw.githubusercontent.com/kubearmor/KubeArmor/main/examples/wordpress-mysql/wordpress-mysql-deployment.yaml
+```
+
+### 4. Use cases and deploying Security Policy
+
+<img src="../.gitbook/assets/demo-use-cases.png" width="700" height="370" />
+
+<details>
+  <summary>
+    <b>Use case 1: Audit access to sensitive data paths</b>
+  </summary>
+
+MySQL uses `/var/lib/mysql` path as the default data directory. An audit policy will generate an alert for each operation associated with the resource i.e. a file or binary, specified in the policy. The details logs about the event can help in identifying unauthrized access and potential security breaches. We'll Create a KubeArmor Security policy to audit access to `/var/lib/mysql` folder path recursively.
+
+**Before applying any policy**
+
+There will'be no alert generated by the KubeArmor for any access to the `/var/lib/mysql` directory or subdirectories.
+
+```
+$ POD_NAME=$(kubectl get pods -n wordpress-mysql -l "app=mysql" -o jsonpath='{.items[0].metadata.name}') && kubectl -n wordpress-mysql exec -it $POD_NAME -- bash
+
+root@mysql-6c6fcdccf-v6qhz:/# touch /var/lib/mysql/test
+root@mysql-6c6fcdccf-v6qhz:/# ls /var/lib/mysql/
+auto.cnf     ib_logfile1  mysql		      test    wordpress
+ib_logfile0  ibdata1	  performance_schema
+```
+
+```
+$ karmor log
+local port to be used for port forwarding kubearmor-relay-xxxxxxxxx-xxxxx: 32767
+Created a gRPC client (localhost:32767)
+Checked the liveness of the gRPC server
+Started to watch alerts
+```
+
+we can see that the file `test` has been created in the `/var/lib/mysql` directory and no alert was generated for this event by the KubeArmor.
+
+#### **Applying [security policy](../examples/wordpress-mysql/security-policies/ksp-mysql-audit-dir.yaml) to audit all access to `/var/lib/mysql` directory**
+
+```yaml
+# ksp-mysql-audit-dir.yaml
+
+apiVersion: security.kubearmor.com/v1
+kind: KubeArmorPolicy
+metadata:
+  name: ksp-mysql-audit-dir
+  namespace: wordpress-mysql
+spec:
+  severity: 5
+  selector:
+    matchLabels:
+      app: mysql
+  file:
+    matchDirectories:
+      - dir: /var/lib/mysql/
+        recursive: true
+  action: Audit
+```
+
```

Comment:
open a new terminal session as the current one is now executing the karmor log process

Reply:
we are mentioning it in `Start Observing Alerts` section